azure-container-apps/README.md

```diff
@@ -0,0 +1,371 @@
+
+- For standard deployments, the service will host your SCIM bridge for ~$16 USD/month (as of January 2024). Container Apps pricing is variable based on activity, and you can learn more on [Microsoft's pricing page](https://azure.microsoft.com/en-us/pricing/details/container-apps/).
+- You don't need to manage a DNS record. Azure Container Apps automatically provides a unique one for your SCIM bridge domain.
+- Azure Container Apps automatically handles TLS certificate management on your behalf.
+- The SCIM bridge can be deployed directly to Azure from the Portal using this guide or via the Azure Shell or command line tools in your local terminal using the [support guide](https://support.1password.com/scim-deploy-azure/). There is no requirement to clone this repository.
+
+**Table of contents:**
+
+- [Before you begin](#before-you-begin)
+- [Step 1: Configure the `scimsession` credentials](#step-1-configure-the-scimsession-credentials)
+- [Step 2: Create the container app](#step-2-create-the-container-app)
+- [Step 3: Configure and deploy your SCIM bridge](#step-3-configure-and-deploy-your-scim-bridge)
+- [Step 4: Connect your identity provider](#step-4-connect-your-identity-provider)
+- [Update your SCIM Bridge](#update-your-scim-bridge)
+- [Appendix: Resource recommendations](#appendix-resource-recommendations)
+- [Get help](#get-help)
+
+## Before you begin
+
+Before you begin, familiarize yourself with [PREPARATION.md](/PREPARATION.md) and complete the necessary steps there. You'll also need the following:
+
+- An Azure account with permissions to create a Container App.
+
+> [!NOTE]
+> If you don't have a Azure account, you can sign up for a free trial with starting credit: https://azure.microsoft.com/en-us/free/
+
+### In this folder
+
+- [`aca-op-scim-bridge.yaml`](./aca-op-scim-bridge.yaml): a Container App configuration file for 1Password SCIM Bridge that you can use when you deploy via the command line.
+
+## Step 1: Configure the `scimsession` credentials
+
+To connect the SCIM bridge with your 1Password account, you'll need to save the `scimsession` credentials as a secret variable in the Container App you create. These credentials need to be Base64-encoded to pass them into the environment, but they're saved as a plaintext file when you download them or save them in your 1Password account during the setup.
+
+> [!IMPORTANT]
+> You'll need to update the path to the `scimsession` file you saved during the preparation steps.
+
+Use the following command for your shell to get the Base64 encoded contents of your `scimsession` file:
+
+- **Bash**:
+
+  ```bash
+  cat ./scimsession | base64
+  ```
+- **PowerShell**:
+
+  ```pwsh
+  [Convert]::ToBase64String([IO.File]::ReadAllBytes((Join-Path $PWD.Path 'scimsession')))
+  ```
+
+Copy the output value to your clipboard to use it in the deployment, then save the encoded value in your 1Password account for future use:
+
+1. Find the "scimsession file" item in your 1Password account.
```

Comment:
This assumes they followed the steps in preparation.md, and for this step the one that matters is to [prepare their 1Password account](https://github.com/1Password/scim-examples/blob/main/PREPARATION.md#prepare-your-1password-account). I can make a note of this for future improvement to that article, which should have stepped headers instead of the current ones that don't imply linear following.